src/test/run-pass/generator/too-live-local-in-immovable-gen.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(generators)]
```

Comment:
Can you add a comment explaining what this test is testing? My *guess* is that it's a local variable that wouldn't have to be considered live (because it is borrowed only after the yield)?

Reply:
Or maybe I'm just confused.

Reply:
I added a comment here.